events/OWNERS

```diff
@@ -1,12 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-reviewers:
-  - parispittman
-  - tpepper
 approvers:
   - parispittman
   - mrbobbytables
-  - guineveresaenger
```

Comment:
@guineveresaenger 

Would you like to remain as an approver in the events owners file?